code-challenges.test.js

```diff
@@ -23,17 +23,52 @@ var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
-
-
-
+describe("When arrayShuffler is called", () => {
+  it("return ['yellow', 'blue', 'pink', 'green'] if ['purple', 'blue', 'green', 'yellow', 'pink'] is called",() => {
+    //arrange
+    const argument = colors1
+    //act
+    const actualResult = arrayShuffler(argument)
+    //assert
+
+    let sortedActualResult = actualResult.sort()
+
+  expect(sortedActualResult).toEqual(['blue','green','pink','yellow'])
```

Comment:
💯  for using a different expected outcome array and finding a way to assert on remaining elements. However something to consider is that this test would still pass if you remove the shuffle logic from your function